test/path/mutt_path_pretty.c

```diff
@@ -26,17 +26,40 @@
 #include <stddef.h>
 #include <stdbool.h>
 #include "mutt/lib.h"
+#include "test_common.h"
 
 void test_mutt_path_pretty(void)
 {
-  // bool mutt_path_pretty(char *buf, size_t buflen, const char *homedir);
+  // bool mutt_path_pretty(struct Buffer *path, const char *homedir);
 
   {
-    TEST_CHECK(!mutt_path_pretty(NULL, 10, "apple", true));
+    TEST_CHECK(!mutt_path_pretty(NULL, "/apple", true));
   }
 
+  // test replacement
```

Comment:
I've looked at the [code coverage](https://neomutt.org/dev/analysis/coverage) and there are a couple of negative tests we could add:
- Path doesn't match `homedir`
- Path matches `homedir` but differs, e.g. `/home/john` vs `/home/johnsmith`

Reply:
Good call. Thanks for pointing me to the coverage docs. Done ✅